src/models/StakeRequest.ts

```diff
@@ -39,6 +39,8 @@ export default class StakeRequest extends Comparable<StakeRequest> {
 
   public stakerProxy?: string;
 
+  public blockNumber?: BigNumber;
```

Comment:
why this is an optional parameter? It shouldn't be, in my opinion.

Reply:
Changes done!